cue/data/gen.cue

```diff
@@ -10,7 +10,7 @@ Family: scuemata.#Family & {
                 // TODO must isolate or remove identifiers local to a Grafana instance...?
                 id?: number
                 // Unique dashboard identifier that can be generated by anyone. string (8-40)
-                uid: string
+                uid?: string
```

Comment:
for post dashboard json, uid is not filled in the request, so the validation was failed. need uid to be optional